.prettierrc.js

```diff
@@ -1,7 +0,0 @@
-module.exports = {
-  bracketSpacing: false,
-  jsxBracketSameLine: true,
-  singleQuote: true,
-  trailingComma: 'all',
-  arrowParens: 'avoid',
-};
```

Comment:
I had a lot of issues with the formatDocument. I thought that if I deleted this file it would stop warning about any incorrect indentation. But it didn't help. Should I return this file?

Reply:
Is it ok to create a new js file named "prettierrc" and paste these configurations there? Or is it more complicated to recover this file?